solutions/cses-1084.mdx

```diff
@@ -54,3 +72,65 @@ int main(){
 	return 0;
 }
 ```
+
+</CPPSection>
+
+<JavaSection>
+
+Note: Java will TLE on test case #17.
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class Apartments {
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt();
+		int m = io.nextInt();
+		int k = io.nextInt();
+
+		int[] applicants = new int[n];
+		int[] apartments = new int[m];
+		for (int i = 0; i < n; i++) {
+			applicants[i] = io.nextInt();
+		}
+		for (int i = 0; i < m; i++) {
+			apartments[i] = io.nextInt();
+		}
+
+		Arrays.sort(applicants);
+		Arrays.sort(apartments);
+
+		int i = 0;
+		int j = 0;
+		int ans = 0;
+		while (i < n && j < m) {
+			// Found suitable apartment
+			if (Math.abs(applicants[i] - apartments[j]) <= k) {
+				i++;
+				j++;
+				ans++;
+			}
+			// Apartment is too small -> increment apartment pointer
+			else if (applicants[i] > apartments[j]) {
+				j++;
+			}
+			// Apartment is too big -> increment applicant pointer
+			else {
+				i++;
+			}
+		}
+
```

Comment:
```suggestion
```

Reply:
I like leaving spacing here to separate variables and methods/static classes

Reply:
Oops I meant to refer to the other suggestion